arduino:
  port: /dev/ttyUSB0
  pin:
    battery: adc0
    gasPedal: adc1
    speed: adc6
    gasPedalSwitch: D19
    keySwitch: D22
    forwardIn: D26
    backwardIn: D27
    speedControlerL: D4
    speedControlerR: D3
    returnControl: D9
    forwardControl: D10
    indicatorInL: D42 #not connected
    indicatorInR: D41 #not connected
    indicatorL: D16
    indicatorR: D17
    lightIn: D40 #not connected
    stop: D39
    light: D25
  
battery:
  voltage: 12
  factor: 3.428
  k: 0.013
  d: 11.7

lane:
  heightmultiplier: 9
  cropmultiplier: 2
  bottommultiplier: 9
  low_threshold: 50
  high_threshold: 150

usb_cam:
  video_device: /dev/video0
  image_width: 640
  image_height: 480
  pixel_format: yuyv
  camera_frame_id: usb_cam
  io_method: mmap
  fov: 50
  flip_image: True

lidar:
  port: /dev/ttUSB1

steering:
  p: 1
  i: 0.1
  d: 0.5

throttle:
  p: 1
  i: 0.1
  d: 0.5

gui:
  version: v 0.2
  winX: 640
  winY: 480
  fullscreen: False